### DISCUSSION TITLE
How to diagnose crashes? Can't find errors

### DISCUSSION THREAD

**Discussion by WineBottles** [bottlesdevs/Bottles#742] [Q&A]
Ever since I've updated my flatpak to 2021.11.14-treviso-1 and higher (up to -4 now) I get random crashes not long after launching a game. Bottles does not tell me it crashed the next time I open it and I do not see errors in journalctl so I don't know how to diagnose. I was hoping updateding to 2021.11.14-treviso-2, -3, -4 would fix it, but it just happened a couple minutes ago.

What happens:
1) I launch Bottles
2) Open my bottle
3) Launch the game
4) Close Bottles after the game launches since I don't need it open anymore
After 10-15 seconds the game closes

This happens seemingly at random. There are times where the game runs fine for a couple hours and other times it just closes right away.

What can I do to diagnose this? Thanks

**Comment by mirkobrombin** [comment]
Please post this in our new forum https://forums.usebottles.com
